app/controllers/api/v1/batch_assignment_overrides_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::BatchAssignmentOverridesController < SharedSecretAuthenticatedApiController
+  include CorsSupport
+
+  def create
+    BatchArbitraryAssignmentCreation.create! create_params.merge(force: true)
+    head :no_content
+  end
+
+  private
+
+  def create_params
+    params.permit(:visitor_id, assignments: [:split_name, :variant, :mixpanel_result, :context])
```

Comment:
separately, we [always pass in 'success' as mixpanel_result](https://github.com/Betterment/test_track_js_client/blob/master/src/assignmentOverride.ts#L45).

we have a longer term goal of removing mixpanel-specific things from the backend, so i'd like to remove it from here as well. we can set the value in `BatchArbitraryAssignmentCreation` since I don't think we're fully ready to rip it out.

Reply:
perfecto! will change.